azure-pipelines-osx.yml

```diff
@@ -30,9 +30,29 @@ jobs:
     inputs:
       command: build
       arguments: --no-restore --configuration Release
+  - task: CmdLine@2
+    inputs:
+      # Remove PathMap properties which ensure deterministic builds, since this
+      # interferes with the coverage analysis. sed -i seems unhappy with the
+      # shell glob but perl doesn't mind.
+      script: 'perl -ni -e "print unless /PathMap/" $(System.DefaultWorkingDirectory)/*/*.csproj'
   - task: DotNetCoreCLI@2
     displayName: 'Test Debug'
     inputs:
       command: test
       projects: 'WalletWasabi.Tests/WalletWasabi.Tests.csproj'
-      arguments: --configuration $(testConfiguration) --filter "UnitTests" --logger "console;verbosity=detailed"
+      arguments: --configuration $(testConfiguration) --filter "UnitTests" --logger "console;verbosity=detailed" --collect:"XPlat Code Coverage"
+      publishTestResults: true
+  - task: CmdLine@2
+    inputs:
+      script: 'ls -la $(Agent.TempDirectory)/*/coverage.cobertura.xml'
+  - task: CmdLine@2
+    inputs:
+      script: 'ls -la $(Agent.TempDirectory)/*/coverage.cobertura.xml'
```

Comment:
Is this duplication correct?

Reply:
oops, i forgot to remove that debugging, fixing